tutorials/roofit/rf711_lagrangianmorph.C

```diff
@@ -0,0 +1,173 @@
+/// \file
+/// \ingroup tutorial_roofit
+/// \notebook -js
+/// Morphing effective field theory distributions with RooLagrangianMorphFunc
+/// A morphing function as a function of one coefficient is setup and can be used
+/// to obtain the distribution for any value of the coefficient.
+///
+/// \macro_image
+/// \macro_output
+/// \macro_code
+///
+/// \date January 2021
```

Comment:
```suggestion
/// \date January 2022
```
A timeless classic :)